ups/dax_obscore.cfg

```diff
@@ -0,0 +1,14 @@
+# -*- python -*-
+
+import lsst.sconsUtils
```

Comment:
`noCfgFile=True` in the SConstruct file?

Reply:
I need to learn what does that mean. 🙂

Reply:
it's what we use in daf_butler to tell sconsUtils not to look for a .cfg file. I think the cfg file is only needed for C++ packages at this point.

Reply:
I tried to figure out what `cfg` file does but failed miserably, so it was just a cargo-culting. I removed cfg file and added the option to SConstruct.

Reply:
See eg https://github.com/lsst/sconsUtils/blob/main/configs/log4cxx.cfg

It tells sconsUtils what libraries and include paths need to be added when building C++ extensions.